graal/backends/core/analyzers/scancode.py

```diff
@@ -28,32 +28,39 @@
 from .analyzer import Analyzer
 
 
+SCANCODE_EXEC = "scancode-toolkit-3.0.0/scancode"
```

Comment:
`scancode-toolkit-3.0.0` this could be removed ? if we keep it like this, the day another release is out we will have to modify the code, what do you think ?

Reply:
Agreed. I'll make the changes